src/scripts/includes/test_flask.py

```diff
@@ -0,0 +1,39 @@
+# Imports built modules and flask.
+
+import __builtin__
+import imp
+import marshal
+import os
+import sys
+import traceback
+import collections
+import time
+import math
+import binascii
+import cStringIO
+import _random
+import _functools
+import cPickle
+import _struct
+import select
+import array
+import unicodedata
+import _ssl
+import _hashlib
+import _io
+import datetime
+import zlib
+
+
+import flask
```

Comment:
That is... a lot of imports. Are they all necessary for this or were you just testing some of them. If you were just testing some of them I'd split those out into a separate file for cleanliness

Reply:
This _was_ a test python file for flask. Deleted with psycopg2_scripts. The imports are every library that's used by flask, which is useful to detect if there are any singular libraries broken.